### DISCUSSION TITLE
feat: Trim copied selections

### DISCUSSION THREAD

**Discussion by crizCraig** [open-webui/open-webui#5117] [General]
**Is your feature request related to a problem? Please describe.**
Copying selected messages in the UI adds multiple newlines

**Describe the solution you'd like**
Trim the text in the same way that hitting the copy button does

**Describe alternatives you've considered**
Hitting the copy button trims newlines

**Additional context**
![image](https://github.com/user-attachments/assets/4433c16b-4e8a-40cb-912c-c50ee29bfa9c)

I'm on Mac Sonoma / Chrome Version 128.0.6613.85 (Official Build) (arm64)

**Comment by tjbck** [comment]
Unable to reproduce this issue, could you verify that wasn’t the LLM response?
